### ISSUE TITLE
Scale font size to maintain optimal characters per line

### ISSUE THREAD

**Issue by iandunn** [WordPress/wporg-news-2021#39]
See https://github.com/WordPress/wporg-news-2021/issues/37#issuecomment-932300306 for background

Figma: https://www.figma.com/file/O9I8Jl7kaUMTX39LlRGbGO/News-Page?node-id=8100%3A35705

Implementation: "modular scale", or maybe something simpler like rems? Do a little research; might be enough to check `twenty*` themes.

**Comment by coreymckrill** [comment]
Possibly relevant: https://richtabor.com/fluid-type-scale-theme-json/

**Comment by iandunn** [comment]
some of these examples may also be relevant: https://developer.mozilla.org/en-US/docs/Web/CSS/clamp()

**Comment by tellyworth** [comment]
@kellychoffman @beafialho another issue that you might be able to help with.

**Comment by kellychoffman** [comment]
This feels a bit out of scope for the initial launch - my suggestion is to choose a single font size for all sizes. @beafialho: what do you think?

**Comment by beafialho** [comment]
> my suggestion is to choose a single font size for all sizes. @beafialho: what do you think?

That sounds ok to me.

On a next stage, it can make sense to use an approach similar to https://developer.mozilla.org/en-US/docs/Web/CSS/clamp(), as @iandunn suggested.

**Comment by iandunn** [comment]
@beafialho, can you add mockups for the bottom banner and release template @ `2000px` ? That'd help me avoid guessing on some things.

**Comment by beafialho** [comment]
Sure @iandunn, I'll work on those and place them here.

**Comment by beafialho** [comment]
Here's the bottom banner and release template at `2000px`: https://www.figma.com/file/O9I8Jl7kaUMTX39LlRGbGO/News-Page?node-id=11980%3A53269

If others are needed please let me know.

**Comment by iandunn** [comment]
Thanks! It looks like after `1320px`, the left/right edge padding starts growing, in addition to the font size. I'm assuming that should be applied to all pages, but let me know if it's specific to the Releases template.

**Comment by beafialho** [comment]
> Thanks! It looks like after 1320px, the left/right edge padding starts growing, in addition to the font size. I'm assuming that should be applied to all pages, but let me know if it's specific to the Releases template.

That's right for most of the pages. Not to be applied to all though (Community has its own behaviour that was addressed in another PR, there's also Podcast, All Posts and the [posts](https://www.figma.com/file/O9I8Jl7kaUMTX39LlRGbGO/News-Page?node-id=8100%3A35708) pages that come to mind at the moment).

I'm happy to mock up all of the pages in large screens so that you're not working in vain. Just a heads up that I'm currently juggling several different projects and could be a little slower than usual.

**Comment by iandunn** [comment]
Here's screencasts of the spacing growing on the Releases category and a single post. If they look good to you, then a list of pages to apply it to is probably good enough for now.

Ignore the font size in them, that's only partially done. The version number delay while resizing is just because of the CSS transition.

* single post: https://cloudup.com/cG3DNhlgEOC
* releases: https://cloudup.com/cs1zdGClg0m

**Comment by beafialho** [comment]
> Here's screencasts of the spacing growing on the Releases category and a single post. If they look good to you, then a list of pages to apply it to is probably good enough for now.

Thanks! I had mocked it up slightly differently for the single posts, but this actually works better now that I saw it. Releases also looks great 👍 

The pages where this behaviour applies:
- Events
- Security
- Development
- General Listing (basically every category page that has the capital letter on the left)

Once I have some time I'll mock up the others and maybe we can address them in a different issue, how does that sound?
